lib/test/TestParser.js

```diff
@@ -49,10 +49,17 @@ class TestParser {
                         "Configuration element is not an object",
                         Util.extractLine(documents[0].configuration));
                 }
-                configuration = documents[0].configuration;
+                configuration = configurationOverride || documents[0].configuration;
+                // eslint-disable-next-line no-console
+                console.log(">>> configuration", configuration);
```

Comment:
this log and the next one are going to show for everything, including the CLI users. We have a debug utility if this is needed (const debug = require("../util/Debug"); )